seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/source/reader/fetcher/SplitFetcher.java

```diff
@@ -234,4 +234,19 @@ private void addTaskUnsafe(SplitFetcherTask task) {
         taskQueue.add(task);
         nonEmpty.signal();
     }
+
+    public void enqueueTask(SplitFetcherTask task) {
```

Comment:
why not direct use https://github.com/apache/seatunnel/blob/c6f627fa38d29f83bccb3e1fb86e8962b64dad4e/seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/source/reader/fetcher/SplitFetcher.java#L108 ?

Reply:
ok, the new method is redundant